cmd/config-host-add.go

```diff
@@ -128,6 +128,9 @@ func mainConfigHostAdd(ctx *cli.Context) error {
 	api := args.Get(4)
 	if api == "" {
 		api = "S3v4"
+		if url == "https://storage.googleapis.com" {
```

Comment:
We have to do this little elaborate check to cater for these conditions. 

- URL is `storage.googleapis.com`  and then no signature type was specified default to `s3v2` which this patch addresses. 
- URL is `storage.googleapis.com` and then signature type was specified by it was specified wrong as `s3v4` where we need to error out indicating that user should use `s3v2` instead in the error message. 

To check if the url is `google` we should just use the helper 

```go
if isGoogle(url) {
    api = "S3v2"
}
```